doc/style-guide.md

```diff
@@ -44,3 +45,52 @@ quaility.
     character limit.
 
 **[top](#table-of-contents)**
+
+## Expressions
+
+  - Expressions in return statements should be surrounded by parenthesis for clarity.
```

Comment:
Since we cannot come up with something better, let's just leave it as is for now until we can come up with something better.  Cool?